http/server.go

```diff
@@ -113,6 +126,7 @@ func (s *Server) Start(port uint) error {
 	r := mux.NewRouter()
 	r.Path("/health").Methods(http.MethodGet).HandlerFunc(s.checkHealthHandler)
 	r.Path("/getQuote").Methods(http.MethodPost).HandlerFunc(s.getQuoteHandler)
+	r.Path("/getQuotePegOut").Methods(http.MethodPost).HandlerFunc(s.getQuotePegOutHandler)
```

Comment:
I think that is a best pratice if we define all pegout routes as pegout/getQuote, pegout/validateSignature.
And the name of the route should be in plural getQuotes because can be more than one quote.